samples/06.f.hybrid-speech/README.md

```diff
@@ -110,18 +110,25 @@ Here is the finished `index.html`:
       <script>
         function createFetchSpeechServicesCredentials() {
           let expireAfter = 0;
-          let lastResult = {};
+          let lastPromise;
 
-          return async () => {
+          return () => {
             if (Date.now() > expireAfter) {
-              const speechServicesTokenRes = await fetch('https://webchat-mockbot.azurewebsites.net/speechservices/token', { method: 'POST' });
+              const speechServicesTokenRes = await fetch(
```

Comment:
do these all match the completed code in their respective readmes?

Reply:
Yap, unless I missed a copy-and-paste.

The previous implementation can fire 2+ network calls if called twice.

The new code only fire 1 network call if called twice.